src/client/jest.config.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+    roots: ['<rootDir>/src'],
+    transform: {
+        '^.+\\.tsx?$': 'ts-jest',
+    },
+    testRegex: '(/__tests__/.*|(\\.|/)(test|spec))\\.tsx?$',
+    moduleNameMapper: {
+        '^@app/(.*)$': '<rootDir>/src/$1',
+        '\\.(css|scss)$': '<rootDir>/styleMock.js',
+    },
```

Comment:
to resolve the aliases that we have. And also, in the second part, I've written an alias for matching `css | scss` files in order to mock them. Jest can't resolve the other files except for javascript files.